src/lib/components/breadcrumbs.svelte

```diff
@@ -77,6 +78,11 @@
         } else newOrgModal.set(true);
     }
 
+    async function loadProjectsFromPromise() {
+        loadedProjects = await selectedOrg.projects;
+        return loadedProjects;
+    }
```

Comment:
mhh, I think we should get rid of the big blocks on the bottom.

lets use a pattern like `async function createBottomSheet(...)` and use it like `$: projectsBottomSheet = createBottomSheet(...)`.

that wait we can just `{#await projectsBottomSheet}`